ec/src/models/short_weierstrass/affine.rs

```diff
@@ -390,47 +369,12 @@ impl<P: SWCurveConfig> Valid for Affine<P> {
 }
```

Comment:
Should I move `check()` into the curve config too?

Reply:
I think for the time being we can keep it outside; for elliptic curves, the checks are basically `is_in_subgroup_assuming_is_on_curve`, and we already specialize that.